app/controllers/users_controller.rb

```diff
@@ -4,7 +4,11 @@ def index
     @users = User.all
   end
 
-  def show; end
+  def show
+    @my_reservations = current_user.reserved_or_borrowed_books
+    @books_reserved_or_taken_from_me = current_user.books
```

Comment:
I am having second thoughts about having a status called `borrowed`. The reason why I say that is because the term `borrowed` is too ambiguous here, because it could mean that the book was reserved, or that the book is actually with the borrower. It's ok the fact that `borrower` represents two things, because we actually need that here e.g. we could do `@borrowed_books = current_user.borrowed_books` instead of `@my_reservations = current_user.reserved_or_borrowed_books`

But that means that we need another status to replace `borrowed`. I would suggest using `:allocated_to_borrower`, to indicate that the book has actually been allocated to the borrower or is with the borrower. 

I know this is just a semantic issue, but I've seen things like these cause a lot of confusion in the past, so I think it's better to change it now while we don't have to refactor too much or have data migrations.